src/layout/layout.ts

```diff
@@ -68,7 +68,7 @@ class Layout implements IComponent {
   // them with real data asynchronously (if it takes too long to do at once) by yielding a thread
   // and returning to it later through a set timeout
   private discoverDom() {
-    let discoverTime = getPageContextBasedTimestamp();
+    let discoverTime = Math.round(getPageContextBasedTimestamp());
```

Comment:
Would it be cleaner to provide a rounded timestamp routine rather than rounding in the client?